File: test_encryption/src/main/java/MessageDTO.java

public class MessageDTO {
    String label;

    public MessageDTO(String label) {
        this.label = label;
    }

    public String getLabel() {
        return label;
    }

    public void setLabel(String label) {
        this.label = label;
    }

    @Override
    public String toString() {
        return "{" + "label='" + label + '\'' + '}';
    }
}

File: Server/src/ServerAPI/DTOs/MessageDTO.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace ServerAPI.DTOs
{
	public class MessageDTO
	{
		public MessageDTO() { }

		public string Value { get; set; }
		public override string ToString()
		{
			return "{" + "value='" + Value + '\'' + '}';
		}
	}
}
